app/models/event.rb

```diff
@@ -17,28 +17,29 @@ def self.received(user)
     end
   end
 
-  attr_reader :data, :repo_url, :repo_name, :created_at, :user
+  attr_reader :data, :repo_url, :repo_name, :created_at, :user, :type
 
   def initialize(data)
     @data       = data
-    @repo_url   = "ttps://github.com/#{data.repo.name}"
+    @repo_url   = "https://github.com/#{data.repo.name}"
     @repo_name  = data.repo.name
     @created_at = data.created_at
     @user = find_user
+    @type = get_type(data)
   end
 
   def find_user
     data.actor.login
   end
 
-  def type
-    #add Delete, Issues
-    types = {"CreateEvent" => lambda {"Create " + data.payload.ref_type.humanize},
+  def get_type(info)
+    types = {"CreateEvent" => lambda {"Create " + info.payload.ref_type.humanize},
      "PullRequestEvent" => lambda {"Pull Request"},
-     "PushEvent" => lambda {"Push"}
+     "PushEvent" => lambda {"Push"},
+     "DeleteEvent" => lambda {"Delete" + info.payload.ref_type.humanize},
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new lambda literal syntax `-> {...}`.<br>Space missing inside {.<br>Space missing inside }.